docs: update managing-eventbus in user-manual (#41)
linkall-labs/docs: user-manual/how-to/managing-eventbus.md

get the eventbus info

Usage:
  vsctl eventbus info [flag]  [flags]

Flags:
      --block             if show all block of segment
      --eventbus string   eventbus to show, use , to separate
  -h, --help              help for info
      --segment           if show all segment of eventlog

Global Flags:
  -C, --config string     the config file of vsctl (default "~/.vanus/vanus.yml")
  -D, --debug             is debug mode enable
      --endpoint string   the endpoints of vanus controller (default "127.0.0.1:8080")
      --format string     the output format of vsctl, json or table (default "table")
```

## List Eventbus

There is a `list` sub-command in the `vsctl eventbus` command to list Eventbuses.

```console
$ vsctl eventbus list -h
list the eventbus

Usage:
  vsctl eventbus list [flags]

Flags:
  -h, --help   help for list

Global Flags:
  -C, --config string     the config file of vsctl (default "~/.vanus/vanus.yml")
  -D, --debug             is debug mode enable
      --endpoint string   the endpoints of vanus controller (default "127.0.0.1:8080")
      --format string     the output format of vsctl, json or table (default "table")
```

## Delete Eventbus

> There is a `delete` sub-command in the `vsctl eventbus` command to delete an Eventbus.

Reviewer: ```suggestion
The `delete` sub-command from `vsctl eventbus` lets you delete an Eventbus.
```
